liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/StartDebugMojoSupport.java

```diff
@@ -540,6 +545,33 @@ protected void copyConfigFiles() throws Exception {
         copyDependencies();
     }
 
+    /**
+     * Merges envProps with special properties found in the install (target) server.env.  We return a clone/copy of
+     * envProps, to which any of a list of special properties found in server.env have been added.  We give precedence
+     * to properties already in envProps.
+     */
+    private Map<String, String> mergeSpecialPropsFromInstallServerEnvIfAbsent(Map<String, String> envProps) throws IOException {
+
+        String[] specialProps = { "keystore_password" };
+
+        Map<String, String> mergedProps = new HashMap<String,String>();
+        
+        // Clone to avoid side effects 
+        for (String key : envProps.keySet()) {
```

Comment:
I think you can simply use the HashMap constructor that takes a Map to create a new Map with everything copied over.